# Controle das Aplicações de Fertilizante

## Informações do experimento

**Cultura:** Tomate (*Solanum lycopersicum*)

**Material genético:** San Marzano

**Área experimental:** 42 m²

**Fertilizante utilizado:** NPK 04-14-08

## Objetivo

Registrar as aplicações de fertilizante realizadas durante o cultivo e verificar a quantidade total aplicada.

In [1]:
# ======================================
# CONTROLE DA ADUBAÇÃO
# ======================================

aplicacoes = [120, 140, 135, 125]

total = sum(aplicacoes)
media = total / len(aplicacoes)

print("RELATÓRIO DE ADUBAÇÃO")
print("-" * 40)

for numero, valor in enumerate(aplicacoes, start=1):
    print(f"Aplicação {numero}: {valor} kg/ha")

print("-" * 40)

print(f"Total aplicado: {total} kg/ha")
print(f"Média por aplicação: {media:.1f} kg/ha")

if total >= 500:
    print("\nPlano de adubação concluído.")
else:
    print("\nNecessária nova aplicação.")

RELATÓRIO DE ADUBAÇÃO
----------------------------------------
Aplicação 1: 120 kg/ha
Aplicação 2: 140 kg/ha
Aplicação 3: 135 kg/ha
Aplicação 4: 125 kg/ha
----------------------------------------
Total aplicado: 520 kg/ha
Média por aplicação: 130.0 kg/ha

Plano de adubação concluído.


# Interpretação

O controle da adubação foi realizado em uma área experimental de 42 m² cultivada com tomate da variedade San Marzano.

Foram registradas quatro aplicações do fertilizante NPK 04-14-08, totalizando 520 kg/ha.

O acompanhamento dessas aplicações permite verificar se o manejo nutricional está sendo executado conforme o planejamento, evitando tanto a deficiência quanto o excesso de nutrientes.

Além disso, manter registros das aplicações facilita o planejamento das próximas adubações e contribui para maior eficiência no uso dos fertilizantes.